.github/workflows/test.yaml

```diff
@@ -63,3 +65,43 @@ jobs:
 
       - name: "Generating tests for '${{ matrix.features }}' scenarios"
         run: devcontainer features test -f ${{ matrix.features }} --skip-autogenerated .
+
+  test-all:
+    if: ${{ github.event_name != 'pull_request' }}
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        features:
+          - fish
+        baseImage:
+          - debian:stable-slim
+          - ubuntu:focal
+          - ubuntu:jammy
+          - mcr.microsoft.com/devcontainers/base:ubuntu
+          - mcr.microsoft.com/devcontainers/base:debian
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: "Install latest devcontainer CLI"
+        run: npm install -g @devcontainers/cli
+
+      - name: "Generating tests for '${{ matrix.features }}' against '${{ matrix.baseImage }}'"
+        run: devcontainer features test --skip-scenarios -f ${{ matrix.features }} -i ${{ matrix.baseImage }} .
+
+  test-all-scenarios:
+    if: ${{ github.event_name != 'pull_request' }}
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        features:
+          - fish
```

Comment:
Is there a way to refactor this so we won't have code duplication? 🤔

Reply:
Yeah, I don't want to write the same thing over and over again if I can avoid it, but I currently believe that the GitHub Actions definition does not allow the use of YAML anchors and aliases, so jobs can't be used in multiple places.

Reply:
There are new ways to reuse code with reusable workflows: https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow, but anchors seem like a better fit for us.